frontend/src/routers/OrgRouter/ApplicationRouter.tsx

```diff
@@ -27,71 +25,63 @@ const ApplicationRouter = ({ integrated }: Props) => {
 	const { page: errorPage } = useErrorSearchContext()
 	usePreloadSessions({ page: page || 1 })
```

Comment:
account settings button does not work (routes to `/w/1/account/auth` but does not show the right page)

<img width="261" alt="image" src="https://user-images.githubusercontent.com/1351531/218531487-ce07a4b0-c020-4602-ae9a-d5fac2f9cdbe.png">

Reply:
Thanks! Fixed in [f08fbfd](https://github.com/highlight/highlight/pull/4203/commits/f08fbfd44b6b43305d6a5da7a4df82cc960a7566)